website/docs/docs/test-terms.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Testing Terms component"
+id: "test-terms"
+pagination_next: null
+pagination_prev: null
+---
+
+This should show a term: <Term id="deploying">Deploying</Term>
+
+This should show a term: <Term id="data-wrangling">Data wrangling</Term>
+
+This should show a term: <Term id="model">Model</Term>
+
+This should show demoTerm: <Term id="demoTerm" />
+
+This should show noDisplayText term id as children and displayText not set: <Term id="noDisplayText" />
+
+This has displayText set but no hoverSnippet set: <Term id="noHoverSnippet" />
+
+This should NOT show a hover snippet (but should still show children): <Term id="demo">Demo</Term>
```

Comment:
Hey @JKarlavige This looks correct, but not sure what you mean by it should still show children?

Reply:
Good question! If the `Term` component is used but the `id` isn't found in `hover-terms.md`, it would still show the children of the component, in this case the text `Demo`. These are just some test cases i added for us to test when reviewing the PR. But i will remove all of these before merge.